webrtc/lib/webrtc/rtp_munger.ex

```diff
@@ -130,7 +131,13 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPMunger do
   @spec can_generate_padding_packet?(t()) :: boolean()
   def can_generate_padding_packet?(rtp_munger), do: rtp_munger.last_marker
 
-  @spec munge(t(), Membrane.Buffer.t()) :: {t(), Membrane.Buffer.t() | nil}
+  @spec munge(t(), Membrane.Buffer.t() | SenderReportPacket.t()) ::
+          {t(), Membrane.Buffer.t() | SenderReportPacket.t() | nil}
+  def munge(rtp_munger, %SenderReportPacket{sender_info: sender_info} = rtcp) do
+    sender_info = Map.update!(sender_info, :rtp_timestamp, &calculate_timestamp(&1, rtp_munger))
+    {rtp_munger, Map.put(rtcp, :sender_info, sender_info)}
+  end
+
   def munge(rtp_munger, buffer) do
     # TODO we should use Sender Reports instead
```

Comment:
Can the TODO from line 142 be removed now?

Reply:
I don't think so. I didn't rewrite the part of code that this todo comment relates to